recipes/python-patch/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "patch" %}
+{% set version = "1.16" %}
+{% set sha256 = "c62073f356cff054c8ac24496f1a3d7cfa137835c31e9af39a9f5292fd75bd9f" %}
+
+package:
+  name: python-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
+
+requirements:
+  host:
```

Comment:
I don't think host is a valid conda-forge requirement. (maybe you meant build?)

Reply:
Thanks! I was already wondering what the problem would be. I fixed this.